crates/cosmos/cosmos-integration-tests/src/contexts/bootstrap_legacy.rs

```diff
@@ -57,6 +61,7 @@ pub struct LegacyCosmosBootstrap {
         Box<dyn Fn(&mut serde_json::Value) -> Result<(), Error> + Send + Sync + 'static>,
     pub comet_config_modifier:
         Box<dyn Fn(&mut toml::Value) -> Result<(), Error> + Send + Sync + 'static>,
+    pub dynamic_gas: Option<DynamicGasConfig>,
```

Comment:
This field can be removed.

Reply:
Done in fef302618de281ede44e9eeafd4ddd53b2b0a994